source/content/quicksilver.md

```diff
@@ -47,9 +47,15 @@ Quicksilver scripts that trigger on the deploy hook operate on the state of the
 
 ## Script Type and Location
 
-Quicksilver currently supports `webphp` scripting, which runs a PHP script via the same runtime environment as the website itself. PHP scripts are subject to the same limits as any code on the platform, like [timeouts](/timeouts), and cannot be batched. In the future we may add additional types. The commands will run in order, and only execute the next when the previous has finished or timed out.
+Quicksilver currently supports `webphp` scripting, which runs a PHP script through the same runtime environment as the website itself. PHP scripts are subject to the same limits as any code on the platform, such as [timeouts](/timeouts). PHP scripts cannot be batched, and will run in order, with each command executing after the previous command has finished or timed out. In the future we may add additional types. 
```

Comment:
I'm not sure, but I don't think we should mention things that we don't currently offer (and are not currently working on) such as "In the future we may add additional types. "

how about?...

```suggestion
Quicksilver currently supports `webphp` scripting, which runs a PHP script through the same runtime environment as the website. PHP scripts are subject to the same limits as any code on the platform, such as [timeouts](/timeouts). PHP scripts cannot be batched, and run continuously and sequentially. Each command executes after the previous command has finished or timed out.
```